pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -142,6 +154,67 @@ func (e *cloudWatchExecutor) getRGTAClient(region string) (resourcegroupstagging
 	return e.rgtaClient, nil
 }
 
+func (e *cloudWatchExecutor) getServiceQuotasClient(region string) (servicequotasiface.ServiceQuotasAPI, error) {
+	sess, err := e.newSession(region)
+	if err != nil {
+		return nil, err
+	}
+
+	return newQuotasClient(sess), nil
+}
+
+func (e *cloudWatchExecutor) getQueue(region string) (chan bool, error) {
+	e.queueLock.Lock()
+	defer e.queueLock.Unlock()
+
+	if queue, ok := e.queuesByRegion[region]; ok {
+		return queue, nil
+	}
+
+	concurrentQueriesQuota, err := e.fetchConcurrentQueriesQuota(region)
+	if err != nil {
+		plog.Info("Could not fetch quota")
```

Comment:
Please make it explicit instead, since when a function returns an error you shouldn't trust the other values it returns:
```suggestion
		plog.Info("Could not fetch quota")
		concurrentQueriesQuota = defaultConcurrentQueries
```